### PR TITLE
add ephemeral smoke test

### PR DESCRIPTION
**Pull request by dpensi**
# Description

smoke test into ephemeral environements on MR 

Fixes CCXDEV-6166

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## Testing steps

check this pipeline, must be green

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -17,23 +17,27 @@
 # --------------------------------------------
 # Options that must be configured by app owner
 # --------------------------------------------
-APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
-COMPONENT_NAME="ccx-smart-proxy"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
-IMAGE="quay.io/cloudservices/ccx-smart-proxy"
+export APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="ccx-smart-proxy"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
```
Comment (ikerreyes) [inline pr_check.sh:21]: I think this should be `insights-results-smart-proxy`

Comment (matysek) [reply pr_check.sh]: 'ccx-smart-proxy' is the old name from saas.yml file.

File: pr_check.sh [hunk 2]
```diff
@@ -17,23 +17,29 @@
...
+# Deploy rbac to an ephemeral namespace for testing
+source $CICD_ROOT/deploy_ephemeral_env.sh
 
-# source $CICD_ROOT/build.sh
-# source $CICD_ROOT/deploy_ephemeral_env.sh
-# source $CICD_ROOT/smoke_test.sh
+# Run somke tests with ClowdJobInvocation
+# component name needs to be re-export to match ClowdApp name (as bonfire requires for this)
+export COMPONENT_NAME="ccx-smart-proxy" 
+source $CICD_ROOT/cji_smoke_test.sh
```
Comment (matysek) [inline pr_check.sh:45]: Is there any reason to use 'cji_smoke_test.sh' and not just 'smoke_test.sh'?

**Review by matysek — APPROVED** [review]

File: pr_check.sh (hunk 1 above)
Comment (ikerreyes) [reply pr_check.sh]: "ccx-smart-proxy" is also the name of the [ClowdApp](https://github.com/RedHatInsights/insights-results-smart-proxy/blob/master/deploy/clowdapp.yaml#L40)

File: pr_check.sh (hunk 2 above)
Comment (ikerreyes) [reply pr_check.sh:45]: "smoke_test.sh'" as been [deprecated](https://github.com/RedHatInsights/bonfire/blob/master/cicd/examples/pr_check_template.sh#L46) in favor of "'cji_smoke_test.sh"

**Review by tisnik — APPROVED** [review]
ok

**Review by juandspy — APPROVED** [review]

**Pull request merged**